newstroke_hershey/convert.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+NewStroke/Hershey to FontoBene conversion script.
+"""
+import os
+from ctypes import *
+
+HEADER = """\
+# This font was automatically converted from StrokeFont to FontoBene.
+# - StrokeFont project:        http://vovanium.ru/sledy/newstroke/en
+# - FontoBene specifications:  https://github.com/fontobene/fontobene
+# - Converter script:          https://github.com/fontobene/fontobene-converters
+#
+# As the StrokeFont is released under the CC0-1.0 license, the converted
+# FontoBene font is released under the same license. CC0 licence text:
+# http://creativecommons.org/publicdomain/zero/1.0/
+
+[format]
+format = FontoBene
+format_version = 1.0
+
+[font]
+name = FontoStroke
+id = fontostroke
+version = 1.0
+author = StrokeFont Developers
+author = FontoBene Developers
+license = CC0-1.0
+letter_spacing = 1.8
+line_spacing = 16
+
+---
+"""
+
+REPLACEMENT_GLYPH = 'F^K[KFYFY[K['
```

Comment:
This is just a square box, right? Alternatively we could draw the diamond + question mark symbol: https://en.wikipedia.org/wiki/Specials_(Unicode_block)#Replacement_character